README.md

```diff
@@ -60,6 +60,7 @@ It has **no ads** and is **absolutely free** to use.
 
 ### Languages
 - English
+- Italiano
```

Comment:
@ZinRicky This file will be updated after new version is released.
Please remove this line for now.